### ISSUE TITLE
Undefined session functions due to CoCart not keeping function names the same as WooCommerce session handler

### ISSUE THREAD

**Issue by cha0ran** [co-cart/co-cart#343]
CRITICAL Uncaught Error: Call to undefined method CoCart_Session_Handler::get_session_data() in /data/html/www/wp-content/plugins/airwallex-online-payments-gateway/includes/controllers/AirwallexController.php:131

```php
$logService->debug('asyncIntent() response', [
    'response' => $response,
    'session' => [
        'cookie' => WC()->session->get_session_cookie(),
        'data' => WC()->session->get_session_data(),
    ],
]);
```

**Comment by seb86** [comment]
Thank you for reporting the issue @cha0ran 

Didn't think these two functions would be used outside of the session handler so I didn't think there was any harm to rename them. Clearly this is a mistake and I will patch soon.

**Comment by seb86** [comment]
`get_session_cookie()` was not renamed only `get_session_data()` was.

**Comment by seb86** [comment]
I think that's the only one. 🐑

**Comment by seb86** [comment]
[Fixed in v3.7.3](https://github.com/co-cart/co-cart/commit/507f364a3e2a34fdbeb315a83b7ddbd00bc6cb04)
